### PR TITLE
chore(frontend): fix import class mismatch. Fixes #5444

### LINKED ISSUE #5444: [frontend] mismatched frontend api client
### Environment

*  How did you deploy Kubeflow Pipelines (KFP)?

*  KFP version: 1.5.0-rc

### Steps to reproduce

1. clone the repo with latest version
2. cd frontend
3. npm run apis
4. git status and get modified files
5. found the definition of `ExperimentStorageState` change into `ApiExperimentStorageState` while the import statements in `frontend/src/pages/ExperimentDetails.tsx`, `frontend/src/components/ExperimentList.tsx`, etc. still `import ExperimentStorageState`, leading to import error

### Expected result
No import error, frontend and backend keep consistent.

### Materials and Reference

---

Impacted by this bug? Give it a 👍. We prioritise the issues with the most 👍.

cc @Jeffwan

### PR DESCRIPTION
**Pull request by algs**
Fixes #5444

### CODE REVIEW

**Comment by Jeffwan** [comment]
@algs Can you type "@googlebot I signed it!" once you signed CLA?

File: frontend/server/src/generated/apis/auth/configuration.ts [hunk 1]
```diff
@@ -4,62 +4,63 @@
...
+ * 
  *
  * NOTE: This class is auto generated by the swagger code generator program.
  * https://github.com/swagger-api/swagger-codegen.git
  * Do not edit the class manually.
  */
 
+
 export interface ConfigurationParameters {
-  apiKey?: string | ((name: string) => string);
```
Comment (Jeffwan) [inline frontend/server/src/generated/apis/auth/configuration.ts:15]: Let's try to minimize the code changes. Can you help format the generate files?

Comment (Jeffwan) [reply frontend/server/src/generated/apis/auth/configuration.ts:15]: Try to run ` npm run format` under `frontend` folder

**Comment by Jeffwan** [comment]
Thanks for the contribution.  This issue help resolves #5444

**Review by Jeffwan — APPROVED** [review]

**Comment by Bobgy** [comment]
Thank you for the fix!
Note I changed PR title to chore, because it's not a user facing bug fix.

/lgtm
/approve

**Pull request merged**